styles/index.scss

```diff
@@ -0,0 +1,51 @@
+:root {
+  font-family: 'Bitter', serif;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+//50 85 25
+.game-wrapper {
+  background-image: url('../images/QuizBackground.png');
+  background-size: cover;
+  height: 100vh;
+  //   width: 100%;
+
+  display: grid;
+  grid-template-columns: 1fr 2fr;
+  //   grid-template-rows: 30% 55% 15%;
+  grid-template-rows: 2fr 4fr 1fr;
+  .header-content {
+    background: burlywood;
+    grid-column: 1/3;
+    grid-row: 1/2;
+  }
+  .game-image {
+    background: cadetblue;
```

Comment:
I usually test using something like: `outline: cadetblue 10px dotted`, just an idea :)

Reply:
Yes, it's a better way :) Just I forgot about the outline... ;)